assets/bindata_dep.go

```diff
@@ -0,0 +1,9 @@
+//+build never,!never
```

Comment:
This doesn't work (bindata is downloaded on every build).
I've used buildtag `tools` for that during the initial port.

Reply:
This just prevents us from importing it at _runtime_. IMO, we should fetch it when pulling in the dependencies (even if we don't need it right now).

Reply:
I don't see any benefit for fetching it on every build but I don't mind it.

Reply:
It _should_ cache it. There's no real benefit to fetching it every build but, this way, we lock the version in. We're also doing this for our test dependencies and golangci-lint.